doc/naming.md

```diff
@@ -42,11 +42,15 @@ Most gRPC implementations support the following URI schemes:
     absolute path).
 
 - `unix-abstract:abstract_path` -- Unix domain socket in abstract namespace (Unix systems only)
-  - `abstract_path` indicates a name in the abstract namespace:
+  - `abstract_path` indicates a name in the abstract namespace.
   - The name has no connection with filesystem pathnames.
   - No permissions will apply to the socket - any process/user may access the socket.
-  - Abstract sockets are represented by a null byte ('\0') as the first character;
-    the dialler will prepend this null. Do not include the null in `abstract_path`.
+  - `abstract_path` cannot contain null bytes.
+    - TODO(#24638): Unix allows abstract socket names to contain null bytes, 
```

Comment:
Please include the full URI of the issue here.

Reply:
Done